content/docs/1.7.0/high-availability/rwx-volume-fast-failover.md

```diff
@@ -20,3 +20,25 @@ For more information, see https://github.com/longhorn/longhorn/issues/6205.
 > kubectl -n longhorn-system delete lease pvc-2ce4e82e-7ccc-46c0-90a8-a141501fbf93
 > ```
 > See, for example, https://github.com/longhorn/longhorn/issues/9093.
+
+### Performance Testing
+
+If your cluster has many RWX volumes, enabling the RWX Volume Fast Failover feature has a high impact on system performance in terms of `API requests`, `RPC rate`, `CPU`, and `memory` usage. Below are key observations from testing:
```

Comment:
```suggestion
According to the benchmark of resource consumption and the investigation of performance impact using 60 RWX volumes, enabling RWX volume fast failover results in an increase in API request rate and RPC rate, as well as a slight increase in CPU and memory consumption.
```

Reply:
Ok good, I'd like to suggest splitting that long sentence up a bit.

```suggestion
The Longhorn project has investigated and benchmarked resource consumption and the impact on performance using 60 RWX volumes. This shows that enabling RWX volume fast turnover results in an increased API request rate and a slight increase in CPU and memory use.
```